builder/builder.go

```diff
@@ -201,83 +201,58 @@ func (b *Builder) SignManifestMoM() error {
 	return nil
 }
 
-// UpdateRepo will fetch the clr-bundles for our configured Clear Linux version
-func (b *Builder) UpdateRepo(ver string, allbundles bool) {
-	// Make the folder to store all clr-bundles version
-	if _, err := os.Stat("clr-bundles"); err != nil {
-		if err = os.Mkdir("clr-bundles", 0777); err != nil {
-			fmt.Fprintf(os.Stderr, "ERROR: Failed to create clr-bundles: %s\n", err.Error())
-		}
-	}
+const (
+	upstreamBundlesBaseDir   = ".mixer/upstream-bundles/"
+	upstreamBundlesVerDir    = "clr-bundles-%s"
```

Comment:
Optional: might worth adding Format or Fmt to the prefix to indicate this is not a dir to be used directly.

Reply:
To which? The format number itself (as opposed to the word) is in name of the directory the bundles get unpacked into.

As is, it ends up as `<dir where mixer is run>/.mixer/upstream-bundles/clr-bundles-1930/bundles/<bundle file>`.  Everything after `upstream-bundles` is a side effect of the structure within the tarball. (Going forward, we probably want a more generic solution to support derivatives that don't package their bundle definitions the same as CLR, but that's a bigger problem to solve for later.)

If you had two copies checked out simultaneously (`prune` as `false`), you'd have, say, `.mixer/upstream-bundles/clr-bundles-1930` and `.mixer/upstream-bundles/clr-bundles-1980` side-by-side.

Going forward (as mentioned in my proposals and in the meeting this morning), there will be other directories inside `.mixer`. The `upstream-bundles` directory is one that will be in the `.gitignore`.

Reply:
Clarifying, I meant a suffix (not prefix! sorry) to the variable name (not directory names in the filesystem), the suggestion was to call upstreamBundlesVerDir something like upstreamBundlesVerDirFormat (the one in the line this comment refers to).

Optional: When in those situations I noticed I end up creating a function to produce the right path for me instead of giving the format out. The calling code looks a bit better.

Reply:
IOW, I was not talking about anything related to the paragraphs 2-4 of your reply ;-)

Reply:
Ohhhhhh, completely misread your comment. Good idea. Fixed in forthcoming push.